packages/esm-patient-medications-app/src/order-basket/order-basket-search.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import OrderBasketSearch from './order-basket-search.component';
+
+const testProps = {
+  encounterUuid: '',
+  onSearchResultClicked: () => {},
+};
+
+describe('OrderBasketSearch: ', () => {
```

Comment:
This test might be one layer more granular than needs to exist. I would just test the whole search component together (box & results).